packages/compiler/src/init/scaffold.ts

```diff
@@ -42,6 +42,11 @@ export interface ScaffoldingConfig {
    */
   libraries: InitTemplateLibrarySpec[];
 
+  /**
+   * Whether to generate a .gitignore file.
+   */
+  includeGitIgnore: boolean;
```

Comment:
```suggestion
  includeGitignore: boolean;
```

thikn would be more consitent?